agent/agentssh/forward.go

```diff
@@ -209,3 +233,14 @@ func directStreamLocalHandler(_ *ssh.Server, _ *gossh.ServerConn, newChan gossh.
 
 	Bicopy(ctx, ch, dconn)
 }
+
+// unlink removes files only.
+func unlink(path string) error {
+	// From os/file_posix.go:
```

Comment:
I read this comment as meaning that `unlink` is implemented in `file_posix.go`, but I think it's just this `EINTR` loop construct --- needs a comment why it's even required, rather than just where it's from.

Reply:
I happened to spot this in stdlib, and added it just-in-case, mainly being cautious. I'll add a comment that `os.Remove` does the same, that should suffice and if someone is interested, they can dive in.